### PR TITLE
LoanApp - JPA

### PR DESCRIPTION
**Pull request by umiidi**

### CODE REVIEW

File: LoanAppJPA/src/main/java/az/growlab/LoanApp/model/Client.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    @OneToOne
+    @JoinColumn(name = "personal_id")
+    private PersonalInformation personalInfo;
+
+    @OneToOne
+    @JoinColumn(name = "loan_id")
+    private LoanInformation loanInfo;
+
+    private String actionStatus;
```
Comment (brhmshrnv) [inline LoanAppJPA/src/main/java/az/growlab/LoanApp/model/Client.java]: better to move special domain and use enums

File: LoanAppJPA/src/main/java/az/growlab/LoanApp/model/Client.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    private String rejectReason;
+    private String finalStatus;
```
Comment (brhmshrnv) [inline LoanAppJPA/src/main/java/az/growlab/LoanApp/model/Client.java]: use enums

**Review by brhmshrnv — CHANGES_REQUESTED** [review]

File: LoanAppJPA/src/main/java/az/growlab/LoanApp/model/Client.java (hunk 1 above)
Comment (umiidi) [reply LoanAppJPA/src/main/java/az/growlab/LoanApp/model/Client.java]: can you write in more detail? I didn't understand "move special domain"

Comment (brhmshrnv) [reply LoanAppJPA/src/main/java/az/growlab/LoanApp/model/Client.java]: move it to Order domain and associate client domain with order domain

**Pull request closed without merging**